v1.1/stop-a-node.md

```diff
@@ -14,6 +14,11 @@ For information about permanently removing nodes to downsize a cluster or react
 
 ### How It Works
 
+- Finishes in-flight requests.  Note that this is a best effort that times out after the duration specified by the `???` cluster setting (1.1.5)
+- Transfers all *range leases* and Raft leadership to other nodes. (1.1.6)
```

Comment:
This still happens as of 1.1.6. In 1.1.5 and earlier, these parts didn't always happen as intended.

Reply:
Great, thanks!  Since the stable web docs show 1.1.6 now, I'll leave this in (minus the version number).